### ISSUE TITLE
Druid: doubled carpets, missing plating under walls, extinguisher accessibility

### ISSUE THREAD

**Issue by mnva0** [new-frontiers-14/frontier-station-14#4350]
## Description
1. Doubled carpet. Front of the ship, southwest corner
2. Missing plating under the two front walls next to the airlocks. I have no idea if this is a problem but it doesn't exactly feel intended
3. Also, the fire extinguisher is apparently inaccessible from the inside without unanchoring and moving the reclaimer. I *also* kind of suspect it's rotated incorrectly (I believe north) because you have to be on the upper part of the tile to access it and it can be accessed from outside of the shuttle

**Reproduction**
1. Look for the things mentioned in the description of this issue

**Screenshots**
<img width="112" height="68" alt="Image" src="https://github.com/user-attachments/assets/f4663539-aa0b-4898-befc-62e313f7bb02" />

<img width="381" height="165" alt="Image" src="https://github.com/user-attachments/assets/ec535cef-c385-4190-ab2d-50e59e9440b1" />

<img width="235" height="295" alt="Image" src="https://github.com/user-attachments/assets/612c1394-0ed7-4908-944b-0718761a74f9" />

<img width="292" height="265" alt="Image" src="https://github.com/user-attachments/assets/6d7a262a-a9f5-4615-8b0a-0a84c1dc28dc" />

<img width="210" height="178" alt="Image" src="https://github.com/user-attachments/assets/a117690d-2b4b-419c-b1ae-0138efa838a6" />

**Additional context**
